uui/components/pickers/PickerInput.tsx

```diff
@@ -178,7 +183,7 @@ function PickerInputComponent<TItem, TId>({ highlightSearchMatches = true, ...pr
         <Dropdown
             renderTarget={ (dropdownProps) => {
                 const targetProps = getTogglerProps();
-                return renderTarget({ ...dropdownProps, ...targetProps });
+                return renderTarget({ id: props?.id, ...dropdownProps, ...targetProps });
```

Comment:
Can we move 'id' prop defining to the targetProps?

Reply:
fixed